testgrid/config.yaml

```diff
@@ -5811,9 +5818,32 @@ dashboards:
     alert_options:
       alert_mail_to_addresses: kubernetes-sig-cluster-lifecycle@googlegroups.com
     alert_stale_results_hours: 24
-    num_failures_to_alert: 4 # Runs every 6h. Alert when it's been failing for 24 hours
+    num_failures_to_alert: 6 # Runs every 2h. Alert when it's been failing for 12 hours
```

Comment:
this was incorrect before - runs every 2 hours.
going to remove these k-a jobs completely eventually - once we resolve politics.